### PR TITLE
Eag 1/feat/header   component

### PR DESCRIPTION
**Pull request by dev-bazz**
# Top Navbar Header
https://linear.app/zuri-project/issue/EAG-1/header-component

# Changes proposed
> Talk about the things you did eg. files changes, dependencies installed e.t.c
- Mostly Worked in the Navbar Component folder
- added an asset folder
# Check List (Check all the applicable boxes)
- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the **Dev** (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.
# Screenshots/ Videos
localhost-3000
[localhost-3000 (1).webm](https://github.com/hngx-org/zuriportfolio-frontend/assets/67967265/843960fe-6773-48f9-ab92-0608f1c66603)

### CODE REVIEW

**Comment by shalomtaiwo** [comment]
@dev-bazz  Please resolve the pending conflict and Use the component buttons created.

**Comment by shalomtaiwo** [comment]
@dev-bazz The conflict is not yet resolved.

**Review by shalomtaiwo — CHANGES_REQUESTED** [review]
Please pull from dev branch and solve the conflicts

**Review by dev-bazz — COMMENTED** [review]
Okay I would fix it

**Review by dev-bazz — COMMENTED** [review]
I have resolved conflit please how will i get the button component so that i can implement it on the header

**Review by dev-bazz — COMMENTED** [review]
I have used the button component but and updated the Zuri logo to an SVG.

However the Button Component is throwing an error saying that the prop"intent" does not exist .

The Button works but just wanted to bring this to your notice

File: components/Navbars/TopBar.tsx [hunk 1]
```diff
@@ -1,7 +1,199 @@
...
+                      <path d="M8.25 22a1.25 1.25 0 100-2.5 1.25 1.25 0 000 2.5z"></path>
+                      <path d="M9 8h12"></path>
+                    </g>
+                  </g>
+                </svg>
+              </div>
+              <span className=" lg:hidden">Cart</span>
+            </div>
+            <div className="justify-center items-center lg:w-auto w-[100%] gap-2 lg:flex-row flex flex-col">
+              <Button className="rounded-lg border-0 bg-green-50 bg-opacity-50" intent={'secondary'} size={'md'}>
```
Comment (dev-bazz) [inline components/Navbars/TopBar.tsx:137]: This is the button Component Added

Comment (shalomtaiwo) [reply components/Navbars/TopBar.tsx:137]: I understand. Please pull from the dev branch one more time. I added the vscode to the gitignore, it should not be there either.

Comment (dev-bazz) [reply components/Navbars/TopBar.tsx:137]: Okay will do that

**Pull request merged**